sql/catalyst/src/main/scala/org/apache/spark/sql/util/ArrowUtils.scala

```diff
@@ -88,6 +90,16 @@ object ArrowUtils {
           fields.map { field =>
             toArrowField(field.name, field.dataType, field.nullable, timeZoneId)
           }.toSeq.asJava)
+      case MapType(keyType, valueType, valueContainsNull) =>
+        val mapType = new FieldType(valueContainsNull, new ArrowType.Map(false), null)
```

Comment:
`valueContainsNull` is for map values have null or not. For FieldType here, I think it is like ArrayType and StructType cases, we should use given nullable parameter.

Reply:
Nice catch. Fixed.

Reply:
yup. i mistakenly reverted this commit, fixed in new revision

Reply:
Seems not yet fixed?

I guess @viirya is asking:

```scala
val mapType = new FieldType(nullable, new ArrowType.Map(false), null)
```

, and I agree with it.

WDYT?

Reply:
Oh, i understood now, my bad i was getting confused with other fields.